hw01/hw01_exploring_penguins.Rmd

```diff
@@ -0,0 +1,46 @@
+---
+title: 'HW1: Exploring Penguins'
+author: "Neslihan Caliskan"
+date: "1/25/2021"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+
+```
+
+## Descriptive Summary of the Dataset
+
+```{r Penguins}
+
+library("palmerpenguins")
+
+palmerpenguins::penguins
+
+##Get a descriptive summary of the penguins dataset
+summary(penguins)
+
+##Check how many columns does penguins dataset has?
+head(penguins)
+
+##Check how many rows does the penguins dataset has?
+nrow(penguins)
+
+Mean_Penguins <- mean(penguins, body_mass_g, na.rm = TRUE)
+```
+
+## Plots
+
+Plotting Bill Length and Bill Depth 
+
+```{r penguins, echo=FALSE}
+library("palmerpenguins")
+
+plot(penguins$bill_length_mm,penguins$bill_depth_mm, main = "Bill Length and Bill Depth", xlab = "Bill Length", ylab= "Bill Depth")
+
+
+```
+
+Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.
```

Comment:
I really enjoyed how you organized your information and how you included the questions that you were asked to answers, which made it very easy to view. Also, it was interesting how you added the extra information on line 46. Great job!